.github/workflows/cd-pipeline.yml

```diff
@@ -87,3 +90,45 @@ jobs:
           VIB_ENV_REGISTRY_USERNAME: "${{ steps.get-registry-credentials.outputs.marketplace_user }}"
           VIB_ENV_REGISTRY_PASSWORD: "${{ steps.get-registry-credentials.outputs.marketplace_passwd }}"
           VIB_ENV_BOSSD_RELEASE_ID: "${{ secrets.BOSSD_RELEASE_ID }}"
+
+  # If the CD Pipeline does not succeed we should notify the interested agents
+  slack-notif:
+    runs-on: ubuntu-latest
+    needs:
+      - vib-publish
+      - update-index
+    if: always()
+    name: Notify unsuccessful CD run
+    steps:
+      - name: Notify in Slack channel
+        if: ${{ needs.vib-publish.result != 'success' || needs.update-index.result != 'success' }}
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: ${{ secrets.CD_SLACK_CHANNEL_ID }}
```

Comment:
Both this envar and `CD_SLACK_BOT_TOKEN` are set at the repo level for `bitnami/charts`, so we may need to add them in this repo still (I can't access the settings and confirm).

Reply:
Thanks for your review, both should be added, see 
![Screenshot 2023-07-21 at 17 49 02](https://github.com/bitnami/containers/assets/13216600/96c6caf6-3632-4f64-8595-22994bbb4fb1)